data/role/jenkins-slave.yaml

```diff
@@ -183,3 +184,23 @@ windows_networks:
     local_ports: '8080'
     remote_ports: "*"
     enabled: true
+psick::windows::envs::envs_hash:
+  'AWS_ENV':
+    value: "%{::ec2_tag_env}"
+    mergemode: clobber
+    user: 'Administrator'
```

Comment:
I think we have to set these variables for all users. So 'user: ' definition have to be removed if possible.